apps/documentation/src/app/components/example/example.ng.html

```diff
@@ -44,34 +44,34 @@
 
   <div class="relative mt-10 w-full flex-1">
     @if (mode() === 'preview') {
-      <div
-        class="flex h-fit min-h-[280px] items-center justify-center rounded-xl border border-zinc-200 bg-zinc-50 p-8 dark:border-zinc-800 dark:bg-zinc-950"
-      >
-        <ng-container [ngComponentOutlet]="component" />
-      </div>
+    <div
+      class="flex h-fit min-h-[280px] items-center justify-center rounded-xl border border-zinc-200 bg-zinc-50 p-8 dark:border-zinc-800 dark:bg-zinc-950"
+    >
+      <ng-container [ngComponentOutlet]="component" />
+    </div>
     }
```

Comment:
FYI, your prettier version _may_ be older than v3 for this de-indentation to happen.

Reply:
Hmm, we are using v3.3, I must look into this to see what would be causing this 🤔 well spotted.

Reply:
Not sure about it, but I think prettier detects external angular templates based on the component suffix. For .ng.html, adding an override in prettierrc should do the trick. I believe this issue was mentioned before in the RFC for the new style guide

Reply:
Ah brilliant! Thanks for letting me know! That's very useful to know!